### PR TITLE
changed styling to make Vitals and Height & Weight full width of cont…

### PR DESCRIPTION
**Pull request by jonathandick**
…ainer

### CODE REVIEW

File: src/summary/documentation/documentation-section.component.tsx [hunk 1]
```diff
@@ -6,12 +6,15 @@ import VitalsCard from "./vitals-card.component";
 
 export default function DocumentationSection(props: DocumentationSectionProps) {
   return (
-    <>
+    <div>
```
Comment (jonathandick) [inline src/summary/documentation/documentation-section.component.tsx]: By leaving this as <>, it wasn't picking up the styling and the two cards remained side by side.

File: src/summary/documentation/dimensions-card.component.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default function DimensionsCard(props: DimensionsCardProps) {
     <SummaryCard
       name="Height & Weight"
       match={props.match}
-      styles={{ flex: 1, margin: ".5rem", maxWidth: "46rem" }}
+      styles={{ margin: ".5rem", width: "45.5rem", maxWidth: "46rem" }}
```
Comment (fatmali) [inline src/summary/documentation/dimensions-card.component.tsx]: @jonathandick will this be responsive across different screen sizes? Setting the width explicitly might cause unwanted behavior on smaller screens.

Comment (jonathandick) [reply src/summary/documentation/dimensions-card.component.tsx]: In my tests, when converting to Iphone, or Ipad in chrome, it seems to be responsive despite the fixed width property. Do you have a suggested alternative approach to this? I could use the css guidance on responsiveness. 

image

**Comment by jonathandick** [comment]
I'm going to close this PR and delete the branch because I'm worried about my git history

**Pull request closed without merging**